contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+You need to complete all five levels to win. All but the first level have multiple failure modes. These are listed – go through them sequentially. Sometimes, you will find you cannot complete a level, and need to pivot your entire company.
+
+Let's press start...
+
+### Level 1 - Find a significant problem work on
+
+//TODO: Graphic for level here
+
+Start by solving a problem you've had yourself. It needs to be a problem you've _actually tried to solve_, otherwise it can't be a significant problem.
+
+The type of idea, _and_ thus the types of users you are targeting, will greatly influence your life. I recommend trying out a bunch of ideas _and then_ figuring out what you like and why.
+
+Here are some pointers:
+
+- **You'll enjoy ideas for users you want to work with.** Early on in PostHog's life, we worked on a tool for sales leaders. We quickly grew frustrated at extroverted people who were positive on calls, but didn't follow through on promises. As a result, we decided to work with customer support, or engineers – two groups we felt would be more reliable to work with, albeit with their own downsides.
+
+- **Known problems are easier to validate,** but your solution must be very different. If you want to build a venture scale business then, in the long run, your solution must be 10x better, or 10x cheaper, than the existing solutions for significant numbers of people to use and/or pay for it.
+
+- **Be careful when validating problems no one else has tackled.** It's much less likely the problem you're interested in is one that others have. 
+
+- **Be wary about building what's trendy.** It's easier to beat unsexy, badly run, non-innovative companies than software companies with VC funding _and_ using the latest technology. Do not be fooled about news articles covering huge fundraises by startups working in these areas – fundraising + press ≠ success. In fact, the trendier the space, the less likely a big round means the company is succeeding.[^1]
+
+- **Conservative buyers demand differentiation.** You could probably get someone to use a new phone company, but you won't get an enterprise to use an uproven CRM. A rip and replace strategy is harder to execute with less differentiation than selling a product to someone who hasn't got anything in place at all.
+
+During this phase, write down every single potential customer conversation. It's easy to get recency bias, and you may want to come back to a detail that didn't seem important at the time. Be thorough and open ended in your questions.
+
+At PostHog, we built a Google Doc over 100 pages long in the first few weeks of doing this. Capture seemingly innocuous details. You never know if you'll re-use your hard-earned research with a different idea later.
+
+Many people spend months or years attempting to validate their idea in advance, and make no progress as a result. You will always get more clarity by doing something rather than hypothesizing forever. 
```

Comment:
```suggestion
Many people spend months or years attempting to validate their idea in advance, and make no progress as a result. You will always get more clarity by building something rather than hypothesizing forever, so don't spend too long at this stage.
```

@ivanagas I think this clarifies the point better. Bottom line: don't faff around! 😉